src/app/static/src/app/components/plots/bubble/BubblePlot.vue

```diff
@@ -393,16 +393,18 @@
                 const sizeIndicatorName = this.indicatorNameLookup[sizeIndicator];
                 const { format, scale, accuracy } = this.colorIndicator!;
                 const { format: formatS, scale: scaleS, accuracy: accuracyS } = this.sizeIndicator!;
+                const stringUpper_value = (upper_value || upper_value === 0) ? upper_value.toString() : "";
```

Comment:
Bit confused about this - see my comment in utils method - don't think these values will ever be 0 based on that (though it should be as 0 may be valid!) Also, why is this not needed for lower values, only for upper?